src/UnitTests/test_quote_prov.cpp

```diff
@@ -158,6 +192,7 @@ static void GetCrlTest()
     TEST_PASSED();
 }
 
+#if defined __LINUX__
 extern void QuoteProvTests()

```

Comment:
I am curious on why this was excluded from Windows tests

Reply:
This was necessary because we don't have full support for local caching in place for the Azure DCAP for windows yet. @msft-chow will be working on that next.